### ISSUE TITLE
Getting a syntax error while executing cml on CLI

### ISSUE THREAD

**Issue by abhishekPurandare-obs** [iterative/cml#960]
I installed CML using

```
sudo npm i -g @dvcorg/cml
```

but getting this error whenever I execute any CML commands.

```
>> cml ci --help
/usr/lib/node_modules/@dvcorg/cml/src/drivers/bitbucket_cloud.js:383
        responseBody?.error?.message || responseBody?.error || responseBody;
                     ^

SyntaxError: Unexpected token '.'
    at wrapSafe (internal/modules/cjs/loader.js:915:16)
    at Module._compile (internal/modules/cjs/loader.js:963:27)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1027:10)
    at Module.load (internal/modules/cjs/loader.js:863:32)
    at Function.Module._load (internal/modules/cjs/loader.js:708:14)
    at Module.require (internal/modules/cjs/loader.js:887:19)
    at require (internal/modules/cjs/helpers.js:74:18)
    at Object.<anonymous> (/usr/lib/node_modules/@dvcorg/cml/src/cml.js:11:24)
    at Module._compile (internal/modules/cjs/loader.js:999:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1027:10)
```

Is there a proper way to test whether the installation has been done correctly?

**Comment by 0x2b3bfa0** [comment]
@abhishekPurandare-obs, please use Node.js 16+

For instance, you can set [`image`](https://support.atlassian.com/bitbucket-cloud/docs/javascript-nodejs-with-bitbucket-pipelines) to  [`node:17-alpine`](https://hub.docker.com/layers/node/library/node/17-alpine/images/sha256-e4c1ed79355cd19e255c26caf6a46734208a41d06fff16bdeb404cc988b7668a?context=explore) for example.

Alternatively, you can use our container images with CML installed; e.g. [`iterativeai/cml`](https://cml.dev/doc/self-hosted-runners#docker-images)

**Comment by casperdcl** [comment]
Surely `npm i` should get an old v14-compatible version though?

**Comment by dacbd** [comment]
> Surely `npm i` should get an old v14-compatible version though? Or just not install at all if on `node<14`?

https://github.com/iterative/cml/blob/6fb4c106957c2f44c0d0d03cdabc276b8068577f/package.json#L32-L34

**Comment by dacbd** [comment]
the optional chaining commit should have also expanded this to exclude older node versions I believe

**Comment by 0x2b3bfa0** [comment]
Yes, it should be `{"node": ">=16.x"}` probably. Alternatively, we may want to consider using Babel and transpiling our code. 🙈

**Comment by dacbd** [comment]
perhaps a v0.14.2 with the correct engine semver and de-publish the 0.14.1 on npm so it's not accidentally downloaded on a <16.x node version?

**Comment by 0x2b3bfa0** [comment]
@abhishekPurandare-obs, “fixed” with [`v0.15.0`](https://github.com/iterative/cml/releases/tag/v0.15.0)

**Comment by abhishekPurandare-obs** [comment]
Okay..Working after upgrade the `nodejs` version to `v16.14.2`. Thank you!
